app/scripts/modules/core/pipeline/config/stages/resizeAsg/aws/resizeAsgStage.html

```diff
@@ -17,6 +17,7 @@ <h4 class="text-center">
         <input type="text" required ng-model="stage.cluster" class="form-control input-sm" />
       </stage-config-field>
     </div>
+    </div>
```

Comment:
This seems either unnecessary or very necessary, in which case the indentation is off on the preceding lines.